app/app.rb

```diff
@@ -5,32 +5,51 @@
 require 'sinatra/activerecord'
 require 'sinatra/cors'
 require_relative './models'
+register Sinatra::Cors # Add this line.
```

Comment:
I don't remember where this came from, I think I was trying to get the header to work? This should probably leave unless we need it.